cmd/notation/cert/generateTest.go

```diff
@@ -68,19 +110,17 @@ func generateTestCert(opts *certGenerateTestOpts) error {
 			CertificatePath: certPath,
 		},
 	}
-	err = addKeyCore(signingKeys, keySuite, isDefault)
+	err = addKeyToSigningKeys(signingKeys, keySuite, isDefault)
 	if err != nil {
 		return err
 	}
 	trust := opts.trust
+	// Add to the trust store
 	if trust {
-		if err := addCertCore(cfg, name, certPath); err != nil {
+		if err := truststore.AddCert(certPath, "ca", name, true); err != nil {
 			return err
 		}
 	}
```

Comment:
```suggestion
            		fmt.Printf("%s: added to the trust-store %s of type ca", name)
	}
```

Reply:
We don't need this line. When `truststore.AddCert` is called, upon successful, `fmt.Printf("Successfully added %s to named store %s of type %s\n", filepath.Base(certPath), namedStore, storeType)` will do the work.